xapian-bindings/python3/.gitignore

```diff
@@ -27,3 +30,8 @@
 /xapian_wrap.d
 /xapian_wrap.h
 /xapian_wrap.stamp
+/xapianletor.py
+/xapianletor_wrap.cc
+/xapianletor_wrap.d
+/xapianletor_wrap.h
+/xapianletor_wrap.stamp
```

Comment:
The two red characters at the end of this line here mean there's no newline character at the end of the final line of the file.

Reply:
fixed in a fixup commit.